src/bun.js/api/server.zig

```diff
@@ -3217,7 +3217,11 @@ fn NewRequestContext(comptime ssl_enabled: bool, comptime debug_mode: bool, comp
                 if (response.init.headers) |headers_| {
                     if (headers_.fastGet(.ContentType)) |content| {
                         needs_content_type = false;
-                        break :brk MimeType.byName(content.slice());
+
+                        var content_slice = content.toSlice(this.allocator);
+                        defer content_slice.deinit();
+
+                        break :brk MimeType.byName(content_slice.slice());
```

Comment:
this is a use-after-free isn't it?

Reply:
yea, I was just pushing from the macOS to finish on window, we need to MimeType.init and free if it is allocated